Sources_v3/Controllers/ChannelListController_Tests.swift

```diff
@@ -231,6 +236,46 @@ class ChannelListController_Tests: StressTestCase {
         
         AssertAsync.willBeEqual(delegate.didChangeChannels_changes, [.insert(channel, index: [0, 0])])
     }
+    
+    // MARK: - Channels pagination
+    
+    func test_loadNextChannels_callsChannelListQueryUpdater() {
+        var completionCalled = false
+        let limit = 42
+        controller.loadNextChannels(limit: limit) { [callbackQueueID] error in
+            AssertTestQueue(withId: callbackQueueID)
+            XCTAssertNil(error)
+            completionCalled = true
+        }
```

Comment:
Smells like we should create some abstraction for this boilerplate, too. It's everywhere.

Reply:
Probably yes. Will think about it when will touch this again :)